wd_demo.au3

```diff
@@ -1263,3 +1264,25 @@ Func SetupOpera($bHeadless)
 	Local $sCapabilities = _WD_CapabilitiesGet()
 	Return $sCapabilities
 EndFunc   ;==>SetupOpera
+
+Func SetupEdgeIEMode() ; this is for MS Edge IE Mode
+	; https://www.selenium.dev/documentation/ie_driver_server/#required-configuration
+	Local $sTimeStamp = @YEAR & '-' & @MON & '-' & @MDAY & '_' & @HOUR & @MIN & @SEC
+	_WD_Option('Driver', 'IEDriverServer.exe') ;
+	_WD_Option('Port', 5555)
```

Comment:
I've noticed something odd in the logs after running the IE driver more than once --

```
    _WD_GetFreePort ==> Not found [13]
    _WD_IsLatestRelease ==> Success [0] : True
    _WD_Startup: OS:	WIN_11 X64 22621 
    _WD_Startup: AutoIt:	3.3.16.1
    _WD_Startup: Webdriver UDF:	0.13.0 (Up to date)
    _WD_Startup: WinHTTP:	1.6.4.2
    _WD_Startup: Driver:	IEDriverServer.exe (32 Bit)
    _WD_Startup: Params:	-log-file="c:\Users\danpo\OneDrive\Documents\GitHub\WebDriver\log\2023-04-20_133006_WebDriver_EdgeIEMode.log" -log-level=INFO -port=5555 -host=127.0.0.1
    _WD_Startup: Port:	5555 (Unavailable)
```

This doesn't appear to affect the driver's functionality. Unsure if this is an issue with IEDriverServer or something is wrong elsewhere.

Reply:
This is the output from netstat --

```
netstat -aon | find "5555"
  TCP    127.0.0.1:5555         127.0.0.1:60325        TIME_WAIT       0
```

It does appear to timeout after a while, so I guess it's NBD.

Reply:
![image](https://user-images.githubusercontent.com/11089482/233463191-8853de38-b233-414c-bfdb-9097bd2fed3f.png)

Reply:
> you mean that this port is blocked ?

Not blocked, but it is detected as "in use" by `_WD_GetFreePort`. 

> maybe IE is still in background ?

Nope, and we've both observed that the port is eventually released after the timeout occurs. AFAIK, this doesn't occur with the other webdrivers. However, I may be able to modify `_WD_GetFreePort` so that it ignores ports in the TIME_WAIT state.

Reply:
```
C:\Windows\System32>netstat -aon | find "5555"
  TCP    127.0.0.1:5555         0.0.0.0:0              LISTENING       3000
  TCP    127.0.0.1:5555         127.0.0.1:52102        ESTABLISHED     3000
  TCP    127.0.0.1:52102        127.0.0.1:5555         ESTABLISHED     26652
```
PID 3000 => IEDriverServer.exe
PID 26652 => AutoIt3_x64.exe

Reply:
Please check this:
```auotit
_WD_Option('Port', _WD_GetFreePort(5555, 5600))
```

Reply:
Take a look on:
[_WD_GetFreePort usage](https://github.com/Danp2/au3WebDriver/pull/454/commits/9e02585fdc28493a4c954eb8c1b347ae8d76ae91)